src/main/signal_handlers.cpp

```diff
@@ -61,10 +60,6 @@ void print_statistics()
 {
   if (pOptions != NULL && pOptions->getStatistics() && pExecutor != NULL)
   {
-    if (pTotalTime != NULL && pTotalTime->running())
-    {
-      pTotalTime->stop();
-    }
```

Comment:
In case you wonder: this has become completely unnecessary (because the timers gracefully handle this case). After all, we only stop this single timer, but do not care about other timers that may still be running elsewhere...